requirements.txt

```diff
@@ -23,3 +23,5 @@ pydicom
 networkx
 click >= 7
 click-option-group >= 0.5, < 0.6
+openpyxl
```

Comment:
At some point in the past, we used in Clinica a (deprecated) package called `xlrd` fo read excel files with python. This package (`openpyxl`) seems to have a better support.  I did a quick search of `xlrd` in the code base but I didn't find anything. Could you please remove it from `requirements.txt`?

Reply:
Indeed `xlrd` has the following disclaimer " Please use openpyxl where you can..."